README.md

```diff
@@ -0,0 +1,290 @@
+integration for multi-tenancy working with nodes driven by the OneView
+drivers for Ironic.
+
+How the driver works
+--------------------
+
+The OneView Mechanism Driver does not reflect all networking operations
+in Neutron to OneView. To identify if a certain Neutron request should
+be fulfilled, the Mechanism Driver checks if the networks and ports are
+related with networks/connections created on OneView.
+
+There are two cases regarding the usage of networks (both properties are
+in the configuration file detailed in the following):
+
+​1. When the user wants to map a Neutron provider network to the
+OneViewUplinkSet, so that every network/port operation on this provider
+network will be reflected (created) on OneView. This mapping is made
+using the uplinkset\_mappings property. In this situation the
+administrator defines comma-separated triples of:
+
+    uplinkset_mappings=<provider_network>:<logical_interconnect_group_uuid>:<uplink_set_name>
+
+These can be related to two types of Uplink Sets:
+
+-   Ethernet Uplink Sets - to support VLAN networks.
+-   Untagged Uplink Sets - to support flat network.
+
+​2. When the user wants Neutron to be aware of a previously created
+network on OneView, so that this network will be mapped onto a provider
+network, and every port/connection operation using that provider network
+will use the mapped network to attach to the connection on OneView. In
+this situation the administrator defines comma-separated pairs of:
+
+    flat_net_mappings=<neutron_provider_network>:<oneview_network_uuid>
+
+In the case of port operations, only Neutron ports related to managed
+networks with the *local\_link\_information* field populated are
+considered. The mechanism driver also uses the information from the MAC
+address of the requested port, to identify the specific NIC of the
+Server Profile where the operation should be executed. This information
+can be directly configured in the Neutron port or passed by the Ironic
+port field *local\_link\_connection*.
+
+The driver also implements a fault tolerance process to guarantee that
+all networks and ports that are present in Neutron are correctly
+reflected in OneView. To ensure that, the verification is executed in
+the startup of the mechanism driver and periodically after the
+initialization. This synchronization process considers the information
+from the configuration file, and the information stored in the OneView
+Mechanism Driver tables present in the Neutron Database.
+
+Considering these assumptions, OneView Mechanism Driver is capable of
+the following:
+
+-   **Create a network** in OneView for each network creation request in
+    Neutron to the physical provider-networks configured in the driver
+    config file.
+-   **Add networks to Uplink Sets** in OneView for the mapped Uplink Set
+    to the physical provider-network in the driver config file.
+
+    > -   Ethernet Uplink Sets are used with vlan typed provider
```

Comment:
Is it supposed to be a Quote? or a nested unordered list?

- Is it
> - This?

- Or is it
  - This?